receiver.py

```diff
@@ -37,13 +37,17 @@ def data_lines(self):
 class Receiver:
 
     def __init__(self, db_type):
+        #Change this
```

Comment:
I don't need to change it....I thought I would need to, but then I changed my mind.